helm/hpcc/templates/thor.yaml

```diff
@@ -119,6 +121,9 @@ data:
 {{ toYaml (omit $thorAgentScope "logging") | indent 6 }}
 {{- include "hpcc.generateLoggingConfig" (dict "root" $ "me" $thorAgentScope) | indent 6 }}
       type: thor
+{{- if not $thor.thorMasterResources -}}
+{{ deepCopy $thor.thorMasterResources | indent 6 }}
```

Comment:
this isn't the right place for the thor manager/master resources setting afaics.
This is the thorAgent resource settings.

The thor manager settings need to be generated into the thormaster-jobspec.yaml

Reply:
thor-agent reads the values from this section and writes it into thormaster-jobspec.  So, it does eventually end up in thormaster-jobspec.yaml as the resources object. It could be written directly into thormaster-jobspec.yaml as the resources object, if we didn't need to override the values from the workunit.